concrete/src/Cache/Adapter/ZendCacheDriver.php

```diff
@@ -109,20 +162,24 @@ protected function internalSetItem(&$normalizedKey, &$value)
      */
     protected function internalRemoveItem(&$normalizedKey)
     {
-        /** @var Cache $cache  */
-        $cache = Core::make($this->cacheName);
-        $item = $cache->getItem('zend/'.$normalizedKey);
+        $cache = $this->getCache();
+        if ($cache === null) {
+            $result = true;
+        } else {
+            $item = $cache->getItem('zend/'.$normalizedKey);
+            $result = $item->clear();
+        }
 
-        return $item->clear();
+        return $result;
     }
 
     /**
-     * Flush the whole storage.
+     * Flush the whole storage (even when the localization system is not ready).
      *
      * @return bool
      */
     public function flush()
     {
-        return Core::make($this->cacheName)->getItem('zend')->clear();
+        return $this->app->make($this->finalCacheName)->getItem('zend')->clear();
```

Comment:
getCache returns the cache only if all the translations files are ready. BTW one may want to clear the cache even before that occurs...